vars/neoDeploy.groovy

```diff
@@ -41,7 +41,7 @@ import static com.sap.piper.Prerequisites.checkScript
      * @mandatory for deployMode=warParams
      */
     'host',
-        /**
```

Comment:
[nit-pick] unrelated

Reply:
I would like to keep this. Can't we go by the Scouts Rule: _Leave the campground cleaner than you found it.5_?